python_modules/dagster-graphql/dagster_graphql/schema/asset_graph.py

```diff
@@ -137,8 +146,15 @@ class GrapheneAssetNode(graphene.ObjectType):
         beforeTimestampMillis=graphene.String(),
         limit=graphene.Int(),
     )
+    assetObservations = graphene.Field(
+        non_null_list(GrapheneObservationEvent),
+        partitions=graphene.List(graphene.String),
+        beforeTimestampMillis=graphene.String(),
+        limit=graphene.Int(),
+    )
     computeKind = graphene.String()
     configField = graphene.Field(GrapheneConfigTypeField)
+    currentLogicalVersion = graphene.String()
```

Comment:
Thoughts on consolidating all of these versioning fields into one object? Something like:
```
class GrapheneAssetNodeVersion(Graphene.ObjectType):
   currentLogicalVersion = ...
   opVersion = ...
   projectedLogicalVersion = ....
```
This way, we can remove the `is_versioned` resolver and just do something like
```
   version = graphene.Field(GrapheneAssetNodeVersion)
```
and if the version field is null, we can assume the asset is not versioned

Reply:
Not a bad idea but for now I'd like to not change this since the associated frontend changes depend on it. However:

- `is_versioned` has been removed because we no longer distinguish between versioned and unversioned assets
- a later PR is going to be adjusting some of the logic around `computeKind` as well as additional logic related to op versions (which will become multi-part rather than restricted to a single value)-- that would be a good time to possibly consolidate the version fields like you suggest